react-example/src/Hooks/useLocalStorage/useLocalStorage.js

```diff
@@ -0,0 +1,44 @@
+import { useState, useEffect} from "react";
+
+function useLocalStorage(itemName, initialValue) {
+    const [error, setError] = useState(false)
+    const [item, setItem] = useState(initialValue);
+
+
+    useEffect(() => {
+      try {
+        const localStorageItem = localStorage.getItem(itemName);
+        let parsedItem;
+        
+
+        if (!localStorageItem) {
+        localStorage.setItem(itemName, JSON.stringify(initialValue));
+        parsedItem = initialValue;
+        }else{
+        parsedItem = JSON.parse(localStorageItem);
+        }
+
+        setItem(parsedItem);
+        setLoading(false);
+      } catch (error) {
+        setError(error)
+      }
+    },[])
+
+
+    const saveItems = newItem => {
+        try {
+        const stringifiedItem = JSON.stringify(newItem);
+        localStorage.setItem(itemName, stringifiedItem)
+        setItem(newItem);
+        } 
+        catch (error) {
+        setError(error)
+        }
+    };
+
+
+    return {item, saveItems, error}
+}
+
+export default useLocalStorage;
```

Comment:
Why don't use `export default function useLocalStorage` in line 3, like we do in the rest of the react code?